### PR TITLE
Add yaml file for plugging in CA certs.

### PR DESCRIPTION
**Pull request by myidpt**
**What this PR does / why we need it**:
This yaml file can be used to help redeploy CA with user-provided certs/keys. The instructions will be shown in istio.io security task. It is a security feature in V0.2.

**Release note**:

```release-note
NONE
```

### CODE REVIEW

**Comment by yutongz** [comment]
/retest
Sorry, the tests were interrupted by cluster updating.

**Comment by myidpt** [comment]
@ldemailly @wattli  PTAL

File: install/kubernetes/plugin-ca-certs.yaml [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+  replicas: 1
+  template:
+    metadata:
+      labels:
+        istio: istio-ca
+    spec:
+      serviceAccountName: istio-ca-service-account
+      containers:
+      - name: istio-ca
+        image: gcr.io/istio-testing/istio-ca:d8928ed10bdbd180cbad39f9a4eaf8a70ddb37a2
```
Comment (ldemailly) [inline install/kubernetes/plugin-ca-certs.yaml]: what's that version ? should be generated from istio.VERSION no ? or be :0.2.11 ?

**Comment by ldemailly** [comment]
shouldn't this be a template like the other yaml ? (until we get rid of the whole stuff)

**Comment by myidpt** [comment]
This file is directly used by users instead of being used as a template to generate other files.
But I don't mind putting it into template/ (maybe I have a misunderstanding about what should go into template/).

**Comment by ldemailly** [comment]
I mean that we should not hardcode shas anywhere (that isn't auto generated/maintained) specially for our own components - if you wanted to temporarily freeze the CA version it should at least be a tag from docker.io and not a sha from gcr.io/testing

**Comment by myidpt** [comment]
Oh I see. Sorry about that. I will update it.

File: install/kubernetes/plugin-ca-certs.yaml (hunk 1 above)
Comment (myidpt) [reply install/kubernetes/plugin-ca-certs.yaml]: Sorry I should put {CA_HUB}/istio-ca:{CA_TAG} here.

**Comment by myidpt** [comment]
@ldemailly @rshriram Can any of you help to merge this PR? Thanks!

**Comment by yutongz** [comment]
Auto merge is disabled now due to repo migration, let me know if we are ready to turn it back on :)

**Comment by andraxylia** [comment]
I saw on slack the monorepo is ready to accept PRs, and all tests now pass, so we can re-enable the auto-merge.

**Comment by ldemailly** [comment]
yesterday I synced the branch and was going to merged it after it passed tests but got distracted and didn't get a notification when it was ready, and it happened again today that it's out of sync, so yes it'd be good to put back the auto merge queue (assuming the tests are really testing)

/lgtm

**Comment by yutongz** [comment]
Great! My concern would be release-branch. The tool is not able to tell the base branch and only auto-merge PRs to master. 
There is a hacker way though. Inject a fake test and make it successes in master and fails in release branches.

**Comment by yutongz** [comment]
https://github.com/istio/test-infra/issues/581

**Comment by rshriram** [comment]
Please don’t enable auto merge. There are enough people to manually push
the big green “Squash and merge” button. We no longer have dependency
issues. A pr that passes tests can just be merged.

**Comment by yutongz** [comment]
@rshriram  OK, I paused for further discussion

**Comment by ldemailly** [comment]
we still, and more than ever as the repo is biggee, need the autorebase and submit and merge queue
that’s one condition to make this work with more developers contending for 1 repo

**Pull request merged**